cmake/modules/ConfigurePyTorch.cmake

```diff
@@ -15,8 +39,42 @@ function(ProbeForPyTorchInstall)
     endif()
     message(STATUS "Found PyTorch installation at ${PYTORCH_PACKAGE_DIR}")
 
-    # PyTorch stashes its installed .cmake files under share/cmake/Torch.
-    set(Torch_ROOT "${PYTORCH_PACKAGE_DIR}/share/cmake/Torch"
-        CACHE STRING "Torch package root")
+    set(Torch_ROOT "${PYTORCH_PACKAGE_DIR}" CACHE STRING
+        "Torch configure directory" FORCE)
+  endif()
+endfunction()
+
+# NpcompConfigurePyTorch
+# Performs post-find configuration of PyTorch flags. Most of this comes down
```

Comment:
post-find is a little terse. "Performs configuration of C++ compiler flags for the npcomp source files after PyTorch has been found"?